release/models/optical-transport/openconfig-transport-types.yang

```diff
@@ -669,6 +675,20 @@ module openconfig-transport-types {
       up to 16 Gb/s signals, including 10 GbE and OTU2";
   }
 
+  identity SFP_DD {
+    base TRANSCEIVER_FORM_FACTOR_TYPE;
+    description
+      "SFP DD pluggable transceiver supporting up to 100 Gb/s signal
+      with 2x50G physical channels.";
```

Comment:
Actually, physical channel corresponds to optical lane. SFP-DD form factor does not confine the implementation of optical lanes. For example, a SFP-DD 100GBASE-DR1 module may have a single 100G optical lane. This problem in description also applies to existing QSFP28 and added DSFP. For added identities since now, we should stop using "physical channels" in description to avoid confusion. "electrical interface" or "electrical lane" is suggested. What do you think? @yawei-yin @ejbrever

Reply:
Can I get a consensus and an example of the new description (using "electrical interface/lane")?

Reply:
How about
```
SFP-DD electrical interfaces will employ 2 lanes that operate up to 25 Gbps NRZ modulation or 56 Gbps PAM4 modulation, providing solutions up to 50 Gbps or 112 Gbps PAM4 aggregate
```

ref: http://sfp-dd.com/

@ejbrever what do you think?

Reply:
Thanks for the example @dplore! I have updated the description according to your suggestion.

Reply:
I think this makes sense now.  LGTM.